pkg/interface/chat/src/js/api.js

```diff
@@ -98,7 +108,7 @@ class UrbitApi {
   }
 
   chatAction(data) {
-    this.action("chat-store", "json", data);
+    return this.action("chat-store", "json", data);
```

Comment:
I need to access the promise to know when the read receipt request completes, so I added these return statements to `api.js`. Maybe everything in this file should be returning it?

Reply:
Agree that seems like good practice for `api.js` files.